### ISSUE TITLE
Allow test suite to be run outside of a Rails instance

### ISSUE THREAD

**Issue by orlitzky** [naitoh/rbpdf#9]
Hello, I'm packaging rbpdf for Gentoo Linux, and we let our users run the test suite when a ruby package is installed. But during this install process, we don't have a Rails instance available!

The test suite for rbpdf is **almost** independent of Rails, but there are a few places where the `Rails` class is used, and a few `require`s that are needed to run outside of a Rails environment. I made some progress on this myself at https://github.com/orlitzky/rbpdf/commit/3b797a2d4da8b615046c859dd0a8bb645c88c70a. With those changes, I can run the tests from a clean git checkout anywhere on the filesystem:

```
$ testrb-2 -Ilib test/*.rb
Loaded suite [test/rbpdf_bidi_test.rb, test/rbpdf_bookmark_test.rb, test/rbpdf_cell_test.rb, test/rbpdf_content_test.rb, test/rbpdf_css_test.rb, test/rbpdf_dom_test.rb, test/rbpdf_font_func_test.rb, test/rbpdf_font_style_test.rb, test/rbpdf_font_test.rb, test/rbpdf_format_test.rb, test/rbpdf_func_test.rb, test/rbpdf_htmlcell_test.rb, test/rbpdf_html_func_test.rb, test/rbpdf_html_test.rb, test/rbpdf_image_rmagick_test.rb, test/rbpdf_image_test.rb, test/rbpdf_test.rb, test/rbpdf_transaction_test.rb, test/rbpdf_viewerpreferences_test.rb, test/rbpdf_write_test.rb, test/test_helper.rb]
Started
...................................................

Finished in 5.278041106 seconds.
--------------------------------------------------------------------------------
51 tests, 474 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
0% passed
--------------------------------------------------------------------------------
9.66 tests/s, 89.81 assertions/s
```

You will probably want a cleaner fix (and I get a warning about ng.png) but I think this should be possible without intrusive changes.

**Comment by naitoh** [comment]
Sorry, I can't run orlitzky@3b797a2.
Because I don't have testrb-2 on CentOS6.

Please show me testrb-2 scripts source code.

**Comment by orlitzky** [comment]
Ah, sorry. It's part of one of our Gentoo packages. We support different versions of everything at the same time, so we have wrappers to force specific versions of things like test-unit:

```ruby
#!/usr/bin/env ruby
gem 'test-unit', '>= 2.5', '<4'
require 'test/unit'
(r = Test::Unit::AutoRunner.new(true)).process_args(ARGV) or
  abort r.options.banner + " tests..."
exit r.run
```

**Comment by naitoh** [comment]
fixed 2b7943ff4267d3ebe0dcdb78fc58c0bc2b80ee20

**Comment by orlitzky** [comment]
I had to add the line,

```ruby
require 'rbpdf'
```

to `test/test_helper.rb` to get the test suite to run. But otherwise, it works great. Thank you!

**Comment by naitoh** [comment]
> I had to add the line,
Thanks your feedback. I merged it.

fixed f0dddae6ef1e5f486e157f5eaf3163e697e66581

**Comment by orlitzky** [comment]
Awesome, thanks again.
